gym/core.py

```diff
@@ -93,28 +101,34 @@ def reset(
         return_info: bool = False,
         options: Optional[dict] = None,
     ) -> Union[ObsType, tuple[ObsType, dict]]:
-        """Resets the environment to an initial state and returns an initial
-        observation.
-
-        This method should also reset the environment's random number
-        generator(s) if ``seed`` is an integer or if the environment has not
-        yet initialized a random number generator. If the environment already
-        has a random number generator and :meth:`reset` is called with ``seed=None``,
-        the RNG should not be reset.
-        Moreover, :meth:`reset` should (in the typical use case) be called with an
+        """Resets the environment to an initial state and returns the initial observation.
+
+        This method can reset the environment's random number generator(s) if :param:`seed` is an integer or
```

Comment:
:param:`seed` not working
![image](https://user-images.githubusercontent.com/15806078/167290055-61b215c4-e9c1-4610-9c98-d304506fab2c.png)

Reply:
It seems that `:param:` doesn't exist https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects